ordered_model/admin.py

```diff
@@ -174,13 +174,22 @@ def get_queryset(cls, request):
         Returns a QuerySet of all model instances that can be edited by the
         admin site. This is used by changelist_view.
         """
-        qs = cls.model._default_manager.get_query_set()
```

Comment:
About this, a simple workaround : just replace the line by :

        try:
            qs = cls.model._default_manager.get_query_set()
        except AttributeError:
            qs = cls.model._default_manager.get_queryset()